### DISCUSSION TITLE
유니티 버전 질문

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-kor#612] [5. Q&A]
안녕하세요 ! ~ Unity Collaborate 이제 사용 할 수 없게 됐으니까 Plastic SCM 로 바꿔려고 했는데 Unity 2020.3.15f1 부터 가능하는데 혹시 Zepeto World Package는 2020.3.15f1에서 사용하면 돼요 ?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. ZEPETO 월드는 Unity 2020.3.9f1 버전에서만 개발 가능합니다. 
https://studio.zepeto.me/kr/guides/installing-unity-and-registering-zepeto-template
